setup.py

```diff
@@ -32,7 +32,7 @@
         'test': [
             'nose',
             'flake8',
-            'openfisca-country-template == 1.2.7',
+            'openfisca-country-template == 1.3.1',
```

Comment:
1.3.0 ? (current master version)

Reply:
`1.3.0` is declared to be compatible only with `'OpenFisca-Core >= 17.1.0, < 18.0'`